lesson4_task2.py

```diff
@@ -0,0 +1,16 @@
+"""
+The valid phone number program:
+
+Make a program that checks if a string is in the right format for a phone number.
+The program should check that the string contains only numerical characters and is only 10 characters long.
+Print a suitable message depending on the outcome of the string evaluation.
+"""
+
+while True:
```

Comment:
Cool solution with possible infinite tries!
Recommendation on enhancement: `continue `can be omitted, here it is unneeded